src/Altinn.App.Core/Features/Maskinporten/IMaskinportenClient.cs

```diff
@@ -12,21 +12,46 @@ public interface IMaskinportenClient
     /// Sends an authorization request to Maskinporten and retrieves a JWT Bearer token for successful requests.
     /// </para>
     /// <para>
-    /// Will cache tokens per scope, for the lifetime duration as defined in the token Maskinporten token payload,
+    /// Will cache tokens per scope, for the lifetime duration as defined in the Maskinporten token payload,
     /// which means this method is safe to call in a loop or concurrent environment without encountering rate concerns.
     /// </para>
     /// </summary>
     /// <param name="scopes">A list of scopes to claim authorization for with Maskinporten.</param>
     /// <param name="cancellationToken">An optional cancellation token to be forwarded to internal http calls.</param>
-    /// <returns>A <see cref="MaskinportenTokenResponse"/> which contains an access token, amongst other things.</returns>
+    /// <returns>A <see cref="JwtBearerToken"/> which contains an access token, amongst other things.</returns>
     /// <exception cref="Maskinporten.Exceptions.MaskinportenAuthenticationException">
-    /// Authentication failed. This could be caused by an authentication/authorization issue or a myriad of other circumstances.
+    /// Authentication failed. This could be caused by an authentication/authorisation issue or a myriad of other circumstances.
     /// </exception>
     /// <exception cref="Maskinporten.Exceptions.MaskinportenConfigurationException">
     /// The Maskinporten configuration is incomplete or invalid. Very possibly because of a missing or corrupt maskinporten-settings.json file.
     /// </exception>
     /// <exception cref="Maskinporten.Exceptions.MaskinportenTokenExpiredException">The token received from Maskinporten has already expired.</exception>
-    public Task<MaskinportenTokenResponse> GetAccessToken(
+    public Task<JwtBearerToken> GetAccessToken(
```

Comment:
It looks strange to me to encode JWT Bearer in the name of this type, since this is based on convention. There's nothing stopping someone from implementing an API accepting this token in some other random header, right? And I'm not sure if it's useful information for the consumer either. The developer still has to check the API documentation on where they expect the access token to be